app/src/main/java/ch/epfl/sweng/studyup/items/DisplayItemActivity.java

```diff
@@ -0,0 +1,48 @@
+package ch.epfl.sweng.studyup.items;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import ch.epfl.sweng.studyup.R;
+import ch.epfl.sweng.studyup.player.Player;
+
+public class DisplayItemActivity extends AppCompatActivity {
```

Comment:
Shouldn't it extends RefreshContext to be consistent?

Reply:
Yes